osu.Game.Rulesets.Osu/Mods/OsuModDeflate.cs

```diff
@@ -15,6 +17,14 @@ public class OsuModDeflate : OsuModObjectScaleTween
 
         public override string Description => "Hit them at the right size!";
 
-        protected override float StartScale => 2f;
+        [SettingSource("Starting size modifier", "The object starting size modifier")]
```

Comment:
doesn't read too well. maybe this is better?

```suggestion
        [SettingSource("Starting Size", "The initial size multiplier applied to all objects.")]
```

Reply:
Tried to making it clear that the number is a modifier and not approach rate, but there shouldn't be any problems. I prefer the "initial size multiplier" one over the "object starting size", although I never figured out where this is shown in-game

Reply:
Upon inspection it doesn't seem to be shown in-game, although it could be (as a tooltip, for example).